recipes/libsafec/all/conanfile.py

```diff
@@ -99,6 +101,14 @@ def generate(self):
             f"--enable-strmax={self.options.strmax}",
             f"--enable-memmax={self.options.memmax}",
         ]
+
+        if is_apple_os(self) and self.settings.arch in ["x86_64", "armv8"]:
+            tc.configure_args += [
+                "--disable-hardening",
+            ]
+            # create a universal binary
+            tc.extra_cflags.extend(["-arch", "arm64", "-arch", "x86_64"])
+            tc.extra_cxxflags.extend(["-arch", "arm64", "-arch", "x86_64"])
```

Comment:
This isn't needed - Conan already passes the correct one (when needed) based on the input settings, and there doesn't seem to be a need to have a universal binary here

Reply:
Absolutely. I do not know why I did not realize this. My fault 😭